scenario-definitions.yaml

```diff
@@ -0,0 +1,30 @@
+---
+products:
+  opensuse-*-github-x86_64:
+    distri: opensuse
+    flavor: github
+    arch: x86_64
+    version: Tumbleweed
+
+machines:
+  64bit:
```

Comment:
I'm inclined to have this machine type explicitly set to something that denotes it comes from GH, so we could use worker classes later, but that's too much early optimization :), I leave it to you Marius (i.e `MACHINE=github`)

Reply:
I don't think that's a good idea. The machine is there to tell what tests need and workers can provide that. That's not related to denote where tests "come from". For that we already set the flavor.